KlayGE/Plugins/Include/KlayGE/OpenGL/OGLRenderView.hpp

```diff
@@ -225,7 +225,6 @@ namespace KlayGE
 		GLenum target_type_;
 		int array_index_;
 		int level_;
-		uint32_t sample_count_, sample_quality_;

```

Comment:
Please keep those two member variables, and use UNREF_PARAM in constructor on them. Render-to-MSAA-texture will be implemented in the future.